app/platform/platform.c

```diff
@@ -184,6 +184,7 @@ int platform_gpio_read( unsigned pin )
 static void ICACHE_RAM_ATTR platform_gpio_intr_dispatcher (void *dummy){
   uint32 j=0;
   uint32 gpio_status = GPIO_REG_READ(GPIO_STATUS_ADDRESS);
+  uint32 now = system_get_time();
```

Comment:
Yes. In particular, in the last build that I did, I got the following addresses:

```
                0x0000000040101a10                system_deep_sleep_local_2
                0x0000000040101ac4                system_get_time
                0x0000000040101adc                system_os_post
                0x0000000040101b10                system_rtc_mem_write
```